services/backend/services/v4/compose.yaml

```diff
@@ -2,3 +2,4 @@ include:
   - path:
       - ./compose.base.yaml
       - ./services/elastic/.${ELASTIC_ENABLED:+./../}compose${ELASTIC_ENABLED:+.elastic}.yaml
+      - ../ldap/.${LDAP_ENABLED:+./v4/}compose${LDAP_ENABLED:+.ldap}.yaml
```

Comment:
Since LDAP is going to be for both BEs, does it make sense to move this to the backend/compose.yaml?

Reply:
Yes it does!